### ISSUE TITLE
Error on startup

### ISSUE THREAD

**Issue by hendisantika** [techmahedy/laravel-ajax-crud#7]
Nice repo.

I am using Laravel 7.x.x.

I have got error when run your repo (Error on startup).

```
Facade\Ignition\Exceptions\ViewException
Route [login] not defined. (View: /Users/hendisantika/PhpstormProjects/laravel-ajax-crud/resources/views/layouts/app.blade.php)
```

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/3713580/90312176-86b84600-df2c-11ea-81fa-1f6153589f41.png">

Any advice?

Thank You

**Comment by techmahedy** [comment]
This is a very simple error. Please create a login route. error will be solved.

**Comment by hendisantika** [comment]
Yes, but It should We don't have to add or modify something after clone the project.
Only run `composer install` and `php artisan migrate`.
